worlds/messenger/Constants.py

```diff
@@ -1,6 +1,8 @@
 # items
 # listing individual groups first for easy lookup
-NOTES = [
+from typing import List
+
+NOTES: List[str] = [
```

Comment:
`["string", ...]` is already `List[str]`, so the proper type can be inferred by looking at the assignment and neither requested type (the hint) nor the assigned value (`[...]`) are complex enough to warrant spelling it out imo, which means type hinting here just adds clutter and an additional import. Not that this has to be fixed, but it's something worth noting: not everything needs a type hint.